packages/terra-functional-testing/lib/services/wdio-selenium-docker-service.js

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable class-methods-use-this */
+const path = require('path');
+const util = require('util');
+const childProcess = require('child_process');
+const Logger = require('../logger/logger');
+
+const exec = util.promisify(childProcess.exec);
+const logger = new Logger({ prefix: 'wdio-selenium-docker-service' });
+
+class SeleniumDockerService {
+  constructor(options = {}) {
+    const { image } = options;
+
+    this.image = image || '3.141.59-20200525';
```

Comment:
Can we be more specifc than image? you don't really get any image, just a selenium image version
```suggestion
    this.seleniumImageVersion = seleniumImageVersion || '3.141.59-20200525';
```

Reply:
based on how the api looks down in the doc, how about this?
```suggestion
    this.imageVersion = imageVersion || '3.141.59-20200525';
```

Reply:
I'm tempted to match these to the ENV, although they don't necessarily need to be.

What do you think about `seleniumDockerVersion`?

My next up order of favorites would be:

`image`

`version`

My personal preference is short names when it's not ambiguous, but I can live with either of the other suggestions. `imageVersion`, `seleniumImageVersion`

Reply:
I like naming it in context of the config:
```js
export.config = {
    // ...
    services: [
        [SeleniumDockerService, {
          seleniumDockerVersion: '3.141.59-20200525',
          seleniumImageVersion: '3.141.59-20200525',
          version: '3.141.59-20200525',
          imageVersion: '3.141.59-20200525',
          image: '3.141.59-20200525',
        }]
    ],
    // ...
};
```
I'd be good with `version` in this context.

Reply:
Renamed to `version`

https://github.com/cerner/terra-toolkit/pull/412/commits/359d9f7c6d9c2a604f058eff2416ce412b9f4edc